cg/services/slurm_upload_service/slurm_upload_service.py

```diff
@@ -36,6 +36,10 @@ def upload(self, upload_command: str, job_name: str, case_id: str):
         slurm_id: int = self.slurm_service.submit_job(job_config)
         self.trailblazer_api.add_upload_job_to_analysis(slurm_id=slurm_id, analysis_id=analysis.id)
 
+    def add_upload_job_to_analysis(self, slurm_id: int, case_id: int) -> None:
+        analysis: TrailblazerAnalysis = self.trailblazer_api.get_latest_completed_analysis(case_id)
+        self.trailblazer_api.add_upload_job_to_analysis(slurm_id=slurm_id, analysis_id=analysis.id)
```

Comment:
Is there a unique constraint on the `slurm_id` column for jobs in trailblazer?

Reply:
No, so there will not be any issues with temporarily creating two upload jobs with the same slurm id before we remove the rsync analyses